weekly/2023/08/07.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Week 31 in Packit
+date: 2023-08-07
+tags:
+  - 2023
+  - August
+  - 2023-August
+---
+
+# Week 31 (August 1st – August 7th)
+
+- Packit's license in RPM specfile is now confirmed to be SPDX compatible. ([packit#2026](https://github.com/packit/packit/pull/2026))
```

Comment:
I don't have a strong opinion whether to mention this in the post, but if we want to do so, let's mention that this was done in all our projects now (packit, ogr, specfile)

Reply:
I was actually thinking about promoting the talk from Flock? But wasn't sure about it